zerver/migrations/0254_userprofile_zoom_token.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.25 on 2019-11-06 22:40
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0253_remove_zoom_video_chat'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='userprofile',
+            name='zoom_token',
+            field=django.contrib.postgres.fields.jsonb.JSONField(default=None, null=True),
+        ),
+    ]
```

Comment:
I guess we may want to convert this to a stupid ZoomAccessToken table, with the 5 fields we need an an index on `zoom_user_id` so we can do deauthorization requests reasonably.